e2e/wdio/core/tests/side-navigation.e2e-spec.ts

```diff
@@ -0,0 +1,92 @@
+import {
+    click,
+    getAttributeByName,
+    getElementArrayLength,
+    isElementDisplayed,
+    getElementClass,
```

Comment:
getElementClass doesn't exist in wdio.ts yet because your PR isn't updated with latest `main`  -- need to follow steps previously sent for creating new branches